packages/vechain/README.md

```diff
@@ -50,6 +50,37 @@ module.exports = {
     }
 };
 ```
+## Multi network support sample `hardhat.config.js`
+
+Multiple network can also be configured to simplify testing and deployments. Networks which are targeting vechain Thor
```

Comment:
Minor: Should we use `VechainThor` to express the blockchain? (same for the occurrences in the other README files)